modules/microshift-deploying-a-load-balancer.adoc

```diff
@@ -0,0 +1,113 @@
+// Module included in the following assemblies:
+//
+// * microshift_networking/microshift-networking.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-deploying-a-load-balancer_{context}"]
+= Deploying a TCP load balancer on a workload
+
+{product-title} offers a built-in implementation of network load balancers. This process uses the node IP address as the external IP address for the `LoadBalancer` service configuration file. 
+
+.Prerequisites 
+
+* You installed the OpenShift CLI (`oc`)
+* Access to the cluster as a user with the cluster-admin role.
+* A cluster installed on infrastructure configured with the OVN-Kubernetes network plugin.
+* The KUBECONFIG environment variable is set.
+
+.Procedure 
+
+. Verify that your pods are running by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -A
+----
+
+. Create a namespace with the following commands. 
++ 
+[source,terminal]
+----
+$ NAMESPACE=nginx-lb-test
+----
++
+[source,terminal]
+----
+$ oc create ns $NAMESPACE
+----
++
+This deploys a test application that verifies the connection.
+
+. Deploy three replicas of the test `nginx` application in your namespace.
++
+[source,terminal]
+----
+$ oc apply -n $NAMESPACE -f https://raw.githubusercontent.com/openshift/microshift/main/docs/config/nginx-IP-header.yaml
+----
+
+. You can verify that the three replicas started successfully by running the following command:
++
+[source,terminal]
+----
+$ oc get pods -n $NAMESPACE
```

Comment:
Does it make sense to run 3 replicas on 1 MicroShift instance? I see the command below is showing that the load balancer does use different backends, but I'm not sure that's a realistic concern for MicroShift users.

Reply:
This deployment is just an Example. Users can decide how many replicas for their applications. And maybe for applications, multiple instances can help to increase the throughput.

Reply:
Going to highlight that these are examples